extensions/ql-vscode/test/vscode-tests/no-workspace/model-editor/suggestion-queries.test.ts

```diff
@@ -0,0 +1,88 @@
+import { createMockLogger } from "../../../__mocks__/loggerMock";
+import type { DatabaseItem } from "../../../../src/databases/local-databases";
+import { DatabaseKind } from "../../../../src/databases/local-databases";
+import { file } from "tmp-promise";
+import { QueryResultType } from "../../../../src/query-server/messages";
+import { QueryLanguage } from "../../../../src/common/query-language";
+import { mockedObject, mockedUri } from "../../utils/mocking.helpers";
+import { Mode } from "../../../../src/model-editor/shared/mode";
+import { join } from "path";
+import type { CodeQLCliServer } from "../../../../src/codeql-cli/cli";
+import type { QueryRunner } from "../../../../src/query-server";
+import { QueryOutputDir } from "../../../../src/local-queries/query-output-dir";
+import { runSuggestionsQuery } from "../../../../src/model-editor/suggestion-queries";
+
+describe("runSuggestionsQuery", () => {
```

Comment:
I mainly added this test so that the new function wouldn't get flagged as deadcode 🙈 It's heavily inspired by the [`runModelEditorQueries` test](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/test/vscode-tests/no-workspace/model-editor/external-api-usage-query.test.ts#L96-L174).